helpers/skype_helper.py

```diff
@@ -0,0 +1,35 @@
+"""
+Helper module for Skype
+"""
+import os
+import sys
+import requests
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Was this really needed? This is somewhat of an anti-pattern given that we use pytest.